frontend/sass/norent/styles.scss

```diff
@@ -6,7 +6,7 @@ $norent-root: "../norent";
 
 $jf-body-family: "Open Sans", "Segoe UI", Tahoma, sans-serif;
 $jf-title-family: "Montserrat Black", sans-serif;
-$jf-title-weight: 900;
```

Comment:
Turns out we actually don't need this variable anymore since we don't have variation among our title weights, just our title typefaces.